parameter/log/include/log/LogWrapper.h

```diff
@@ -71,11 +71,11 @@ class LogWrapper
     /** Class destructor */
     ~LogWrapper()
     {
-        if (!mLog.str().empty()) {
-            if(isWarning) {
-                mLogger.warning(mProlog + mLog.str());
-            } else {
-                mLogger.info(mProlog + mLog.str());
+        auto logfunc = isWarning ? &ILogger::warning : &ILogger::info;
+
+        for (auto& log : mLogs) {
+            if (!log.str().empty()) {
```

Comment:
Are you sure this empty test is still useful ? A stream string is only created if there is things to put in.